core/server/src/main/java/tachyon/metrics/MetricsSystem.java

```diff
@@ -176,10 +176,13 @@ private void registerSinks() {
       String classPath = entry.getValue().getProperty("class");
       if (classPath != null) {
         try {
+
+          Class<? extends Sink> clazz = Class.forName(classPath).asSubclass(Sink.class);
```

Comment:
We do not need it in this simple version. We will need it in the rich version in order to check if csv sink is enabled. I'm going to revert it and add it afterwards